src/components/PullRequests/PullRequest.js

```diff
@@ -30,13 +30,25 @@ export const PullRequest = ({ pullRequest }) => {
         <div>
           <div className={styles.preTitle}>
             <div className={styles.owner}>
-              <img
-                src={pullRequest.repository.owner.avatarUrl}
-                alt={pullRequest.repository.nameWithOwner}
-                className={styles.ownerAvatar}
-              />
+              <a
+                href={pullRequest.repository.owner.url}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <img
+                  src={pullRequest.repository.owner.avatarUrl}
+                  alt={pullRequest.repository.nameWithOwner}
+                  className={styles.ownerAvatar}
+                />
+              </a>
               <p className={styles.repo}>
-                {pullRequest.repository.nameWithOwner}
+                <a
+                  href={pullRequest.permalink}
```

Comment:
Do you think it would make more sense for this one to go to the repo rather than the PR? We already have the PR on the title

Reply:
That was the intention but I must have accidentally undone that change 😂